src/main/java/me/clip/placeholderapi/expansion/PlaceholderExpansion.java

```diff
@@ -46,31 +45,27 @@
    *
    * @return placeholder identifier that is associated with this expansion
    */
-  @NotNull
```

Comment:
Why do you remove the Annotations?
The Strings are required to be non-null and the annotation helps with this in IDEs

Reply:
The whole problem stems from them being null anyways, despite being declared as NotNull. See [pastebin error](https://pastebin.com/B2EL8PEW)

Reply:
Still no need to remove the annotations, since they never caused errors so far.
At most add stuff to check for null and replace with default value or cancel expansion registration.
Removal of the snnotations does nothing here.

Reply:
True I'll give you that.